tomo/protocols/protocol_import_subtomograms.py

```diff
@@ -232,29 +232,44 @@ def _summary(self):
             summary.append("%s imported from:\n%s"
                            % (self._getSubTomMessage(), self.getPattern()))
 
-            summary.append(u"Sampling rate: *%0.2f* (Å/px)" %
-                           self.samplingRate.get())
+            if self.samplingRate.get():
+                summary.append(u"Sampling rate: *%0.2f* (Å/px)" %
+                               self.samplingRate.get())
             if self.hasAttribute('outputSubTomogram'):
                 outputSubTomograms = [getattr(self, 'outputSubTomogram')]
             else:
                 outputSubTomograms = getattr(self, 'outputSubTomograms')
-            i = 1
-            for outputSubTomogram in outputSubTomograms:
-                summary.append(u"File %d" % i)
-                summary.append(u"Acquisition angle max: *%0.2f*" % outputSubTomogram.getAcquisitionAngleMax())
-                summary.append(u"Acquisition angle min: *%0.2f*" % outputSubTomogram.getAcquisitionAngleMin())
-                summary.append(u"Step: *%d*" % outputSubTomogram.getStep())
-                summary.append(u"Angle axis 1: *%0.2f*" % outputSubTomogram.getAngleAxis1())
-                summary.append(u"Angle axis 2: *%0.2f*" % outputSubTomogram.getAngleAxis2())
-                i += 1
+
+            if self.importAcquisitionFrom.get() == self.MANUAL_IMPORT:
+                summary.append(u"Acquisition angle max: *%0.2f*" % self.acquisitionAngleMax.get())
+                summary.append(u"Acquisition angle min: *%0.2f*" % self.acquisitionAngleMin.get())
+                if self.step.get():
+                    summary.append(u"Step: *%d*" % self.step.get())
+                if self.angleAxis1.get():
+                    summary.append(u"Angle axis 1: *%0.2f*" % self.angleAxis1.get())
+                if self.angleAxis2.get():
+                    summary.append(u"Angle axis 2: *%0.2f*" % self.angleAxis2.get())
+            else:
+                i = 1
+                for outputSubTomogram in outputSubTomograms:
```

Comment:
@matiasgarcia91 Can we do something a bit more elegant like this https://stackoverflow.com/questions/522563/accessing-the-index-in-for-loops here?

Reply:
Done, had to force-push the branch due to an issue. Best is to delete de branch locally and do a fresh `git fetch --all` and checkout